pytorch_lightning/trainer/supporters.py

```diff
@@ -104,21 +104,6 @@ def _agg_memory(self, how: str):
                 return getattr(self.memory[:self.current_idx], how)()
 
 
-class Accumulator(object):
```

Comment:
what was the history behind this? why do we have this and the tensor accumulator above?

Reply:
No idea about this one, it predates my presence in the project.

`TensorAccumulator` is used for the progress bar running (loss)

https://github.com/PyTorchLightning/pytorch-lightning/blob/0e45220263f4e2045dfe7f68e3e0eaac0b2033d5/pytorch_lightning/core/lightning.py#L1533